### DISCUSSION TITLE
maa,与excel的兼容性问题

### DISCUSSION THREAD

**Discussion by keepzcc** [MaaAssistantArknights/MaaAssistantArknights#7336] [Ideas]
当maa 4.27版本在后台挂机时，会占用剪切板，导致在使用excel的时候出现，‘’剪切板出现问题，你仍可在此工作薄粘贴内容‘’的报错，关闭maa后恢复正常

**Comment by ABA2396** [comment]
剪切板是全局的来着，感觉是偶发性问题，能复现吗

**Comment by keepzcc** [comment]
后面发现了具体的原因，是mumu模拟器的问题，maa运行的时候会和模拟器一起运行，当时就误以为是maa的问题，后面发现关闭模拟器后excel就恢复了正常
















在 2023-11-30 14:14:31，"uye" ***@***.***> 写道：

剪切板是全局的来着，感觉是偶发性问题，能复现吗
